android/intltest/java/com/facebook/hermes/test/HermesIntlGetCanonicalLocalesTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import android.content.res.AssetManager;
+import android.test.InstrumentationTestCase;
+import android.text.TextUtils;
+import android.util.Log;
+
+import static org.fest.assertions.api.Assertions.assertThat;
+
+import com.facebook.hermes.test.JSRuntime;
+
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+// Run "./gradlew :intltest:preparetest262" from the root to download and copy the test files to the APK assets.
+public class HermesIntlGetCanonicalLocalesTest extends HermesIntlTest262Base {
+
+    private static final String LOG_TAG = "HermesIntlGetCanonicalLocalesTest";
+
+    @Test
+    public void testIntlGetCanonicalLocales() throws IOException {
+        String basePath = "test262-main/test/intl402/Intl/getCanonicalLocales";
+        Set<String> whiteList = new HashSet<>();
+        Set<String> blackList = new HashSet<>(Arrays.asList("Locale-object.js"
+                , "canonicalized-tags.js" // All except one tag (cmn-hans-cn-u-ca-t-ca-x-t-u) passes. icu4j adds an extra 'yes' token to the unicode 'ca' extension!
```

Comment:
Since this affects more than one test, is this added 'yes' something we can easily work around?